client/scripts/views/components/markdown_formatted_text.ts

```diff
@@ -10,6 +10,9 @@ import smartTruncate from 'smart-truncate';
 
 const renderer = new marked.Renderer();
 renderer.link = (href, title, text) => {
+  if (href.contains('://localhost') || href.contains('://commonwealth.im/')) {
+    return `<a href="${href}">${text}</a>`;
+  }
   return `<a target="_blank" href="${href}">${text}</a>`;
```

Comment:
I feel like this whole thing could be simply: 
 `<a ${!href.contains(':commonwealth.im/') && 'target="_blank"'} href="${href}">${text}</a>`

We can ignore the localhost case here